source/includes/environment_config/_40-update.md.erb

```diff
@@ -68,7 +68,7 @@ Update some attributes of an environment.
 
 <p class='http-request-heading'>HTTP Request</p>
 
-`PATCH /go/api/admin/environments/environment_name`
+`PUT /go/api/admin/environments/:environment_name`
```

Comment:
Could you backport this change to older branches since when this API was introduced, to make sure that users who are reading old docs are not tripped up?